testbook/client.py

```diff
@@ -93,8 +93,10 @@ def execute_cell(self, cell, **kwargs) -> Union[Dict, List[Dict]]:
         """
         Executes a cell or list of cells
         """
-
-        if not isinstance(cell, list):
+        if isinstance(cell, slice):
+            start, stop = self._cell_index(cell.start), self._cell_index(cell.stop)
+            cell = range(start, stop + 1)
```

Comment:
`range(start, stop, self._cell_index(cell.step))`

Reply:
Sure.. but would we want to support such executions? It could allow users to execute cells in reverse.

Reply:
It's probably non-sensicle to run backwards, but if you're accepting a slice object it's worse to not properly support the object. If you're going to ignore the step though, then you should raise an exception if it's not an expected value.